src/android/PluginError.java

```diff
@@ -2,18 +2,18 @@
 
 public enum PluginError {
 
-    BIOMETRIC_UNKNOWN_ERROR(-100),
-    BIOMETRIC_AUTHENTICATION_FAILED(-102, "Authentication failed"),
-    BIOMETRIC_HARDWARE_NOT_SUPPORTED(-104),
-    BIOMETRIC_NOT_ENROLLED(-106),
-    BIOMETRIC_DISMISSED(-108),
-    BIOMETRIC_PIN_OR_PATTERN_DISMISSED(-109),
+    BIOMETRIC_UNKNOWN_ERROR(-100, "An unknown error occurred."),
+    BIOMETRIC_AUTHENTICATION_FAILED(-102, "Authentication failed."),
+    BIOMETRIC_HARDWARE_NOT_SUPPORTED(-104, "Biometrics is not supported on this hardware."),
+    BIOMETRIC_NOT_ENROLLED(-106, "Biometrics is not configured."),
+    BIOMETRIC_DISMISSED(-108, "Biometrics was dismissed by user."),
+    BIOMETRIC_PIN_OR_PATTERN_DISMISSED(-109, "Biometrics was dismissed by user."),
```

Comment:
Shouldn't this one be "Biometric pin or pattern prompt was dismissed by the user."?

Reply:
Good point, better to have a distinction between the two similar errors.

Reply:
In actuality, this error is only returned when trying the fallback Pin or Pattern - not biometrics - am going to update it like that.

Reply:
Also, once there is a stable library for the fix in https://github.com/OutSystems/cordova-plugin-fingerprint-aio/tree/fix/android10-use-pin - That error is no longer returned (removed it from the code in that branch just now)